cluster/images/conformance/run_e2e.sh

```diff
@@ -39,23 +39,36 @@ saveResults() {
 # We get the TERM from kubernetes and handle it gracefully
 trap shutdown TERM
 
-ginkgo_args=(
-    "--focus=${E2E_FOCUS}"
-    "--skip=${E2E_SKIP}"
-    "--noColor=true"
-)
-
+ginkgo_args=""
 if [[ -n ${E2E_DRYRUN:-} ]]; then
     ginkgo_args+=("--dryRun=true")
 fi
 
+GINKGO_NODES=1
 case ${E2E_PARALLEL} in
-    'y'|'Y')           ginkgo_args+=("--nodes=25") ;;
-    [1-9]|[1-9][0-9]*) ginkgo_args+=("--nodes=${E2E_PARALLEL}") ;;
+    'y'|'Y')           GINKGO_NODES=25 ;;
```

Comment:
WDYT about this for parity (y or numbers seems weird to me)
```suggestion
    'n'|'N')           GINKGO_NODES=1 ;;
    'y'|'Y')           GINKGO_NODES=25 ;;
```

Reply:
@dims WDYT?
this reflects in the YAML.

Reply:
side question here: isn't the default of 25 for parallel a bit too much?